bin/dependency_validator.dart

```diff
@@ -22,6 +22,7 @@ final ArgParser argParser = new ArgParser()
   ..addFlag('verbose', defaultsTo: false)
   ..addOption('ignore', abbr: 'i', allowMultiple: true, splitCommas: true)
   ..addOption('exclude-dir', abbr: 'x', allowMultiple: true, splitCommas: true)
+  ..addFlag('fatal-pins', defaultsTo: true)
```

Comment:
This isn't documented anywhere, and can't be discovered by consumers without reading the source code.

This should be added to the readme. This might also be a good opportunity to fold in work to do the `--help` flag: https://jira.atl.workiva.net/browse/AF-137, if you're feeling ambitious!

Reply:
I'd rather do that in a follow-up PR just for separation of concerns.

Reply:
I'm fine with that, so long as we do it soon, before consumers need to figure out how to use the flag. If it's going to take a while to get to, we should just stick a line in the README.

Reply:
On second thought I added it here: https://github.com/Workiva/dependency_validator/pull/28

I can change the base of that PR to this one and merge it directly here if you'd like.